src/components/landing/pages/LandingContainer.js

```diff
@@ -3,13 +3,16 @@ import RenderLandingPage from '../components/RenderLandingPage';
 import LandingHeader from '../components/LandingHeader';
 import Footer from '../../common/Footer';
 import '../../../antD/styles/landing.css';
+import LandingHFCRS from '../../common/layoutComponents/LandingHFCRS';
 
-function LandingContainer({ LoadingComponent }) {
+function LandingContainer() {
   return (
     <div className="page page-layout">
-      <LandingHeader />
-      <RenderLandingPage />
-      <Footer />
+      <LandingHFCRS
+        HeaderComponents={<LandingHeader />}
+        FooterComponents={<Footer />}
+        ContentComponents={<RenderLandingPage />}
```

Comment:
Is this working for you, or is it rendering a sidebar anyway? I am going to do a pr adding a component for antD layout without a sidebar at some point tonight (2/3/21). If  this is rendering a sidebar for you, you can change and use that layout component after I push it up. Otherwise this looks good to me, so I am approving it.